README-FIPS.md

```diff
@@ -111,14 +115,17 @@ We use OpenSSL 3.1.0 here, (but you could also use the latest 3.0.X)
     $ make
 
 Use the OpenSSL FIPS provider for testing
----------------------------------------------
+-----------------------------------------
 
 We do this by replacing the artifact for the OpenSSL 3.1.0 FIPS provider.
 Note that the OpenSSL 3.1.0 FIPS provider has not been validated
 so it must not be used for FIPS purposes.
 
     $ cp ../openssl-3.0.0/providers/fips.so providers/.
     $ cp ../openssl-3.0.0/providers/fipsmodule.cnf providers/.
+    // Note that for OpenSSL 3.0 that the `fipsmodule.cnf` file should not
+    // be copied across multiple machines if it contains an entry for
```

Comment:
The security policy is somewhat stronger -- as in _never_.
Still, this meets the intent.